.github/workflows/dalgo-ci.yml

```
@@ -88,4 +92,4 @@ jobs:
 
       - name: Check coverage percentage
         run: |
-          coverage report --fail-under=70
\ No newline at end of file
+          uv run coverage report --fail-under=70
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)